app/views/kudo_mailer/batch_kudo_notification.html.erb

```diff
@@ -1,10 +1,10 @@
 <% content_for :message do %>
 
   <% @commentables.each_with_index do |commentable, index| %>
-  
-    <% givers_list = @kudo_givers["#{commentable.class.name}_#{commentable.id}"] %>
-    <%= givers_list.each_with_index.map {|g,i| style_bold((g == "a guest" && i == 0) ? g.capitalize : g)}.to_sentence.html_safe %> 
-    left kudos on
+    <% givers_list = @kudo_givers["#{commentable.id}"] %>
+    <%= if ! givers_list.nil? then givers_list.each_with_index.map {|g,i| style_bold(( i == 0) ? g.capitalize : g)}.to_sentence.html_safe end  %>
+
+    <%=  t 'mailer.kudos.leftkudos' %> 
```

Comment:
The translatable structure here is list of givers + "left kudos on" + title of work kudosed. There might be languages requiring text before the givers or after the title. Interpolation to the rescue (again).